src/main/java/io/candydoc/ddd/extract_domain_vocabulary/ReflectionsDomainVocabularyFinder.java

```diff
@@ -161,11 +161,24 @@ private static Set<FullName> dependenciesOf(Class<?> domainModel, Set<Class<?>>
     return dependenciesOfTacticalPattern(domainModel);
   }
 
+  private static void addSuperClassesDeclaredFieldsIfAny(
+      Class<?> domainModel, Set<Class<?>> dependencies) {
+    Class<?> domainModelSuperClass = domainModel.getSuperclass();
```

Comment:
What happen if the superclass has a superclass ? And what if the superclass of the superclass has a superclass ?

Reply:
This function is recursive and can easily handle a superclass of a superclass of a superclass ... of a superclass